components/properties/src/props.rs

```diff
@@ -284,6 +289,26 @@ impl BidiClass {
     pub const PopDirectionalIsolate: BidiClass = BidiClass(22);
 }
 
+/// TODO: documentation for Enumerated property Bidi_Paired_Bracket_Type
+#[derive(Copy, Clone, Debug, Eq, PartialEq, Ord, PartialOrd)]
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
+#[cfg_attr(feature = "datagen", derive(databake::Bake))]
+#[cfg_attr(feature = "datagen", databake(path = icu_properties))]
+#[allow(clippy::exhaustive_structs)] // newtype
+#[repr(transparent)]
+#[zerovec::make_ule(BidiPairedBracketTypeULE)]
```

Comment:
Nit: I think this does not need to be a ULE. Let's not add ULE impls unless we need them.

Reply:
Yeah, it's necessary to be able to make CPTs for datagen. That said, we _could_ `cfg_attr(feature = "datagen)` this, or just use `CPT<u8>` in datagen.